internal/controller/observability/collector.go

```diff
@@ -125,6 +132,20 @@ func GenerateConfig(k8Client client.Client, spec obs.ClusterLogForwarder, resour
 	return generatedConfig, err
 }
 
+func generateSecretReaderScript(k8Client client.Client, outputs internalobs.Outputs, namespace string) (config string, err error) {
+	var secrets []corev1.Secret
```

Comment:
This is already done early in the controller https://github.com/openshift/cluster-logging-operator/blob/master/internal/controller/observability/clusterlogforwarder_controller.go#L60

Reply:
yes, but that Map contain also secret from Inputs, we don't need them in this case